Cesium3DTilesSelection/test/TestTilesetSelectionAlgorithm.cpp

```diff
@@ -600,20 +601,26 @@ TEST_CASE("Test additive refinement") {
     {
       ViewUpdateResult result = tileset.updateView({viewState});
 
-      // root is rendered first
+      const std::vector<Tile*>& ttr = result.tilesToRenderThisFrame;
+      REQUIRE(ttr.size() == 8);
+
+      // root is done loading and rendered.
       REQUIRE(root->getState() == TileLoadState::Done);
       REQUIRE(!doesTileMeetSSE(viewState, *root, tileset));
       REQUIRE(root->getChildren().size() == 1);
+      REQUIRE(std::find(ttr.begin(), ttr.end(), root) != ttr.end());
 
-      // root's children don't have content loading right now, so only root get
-      // rendered
+      // root's child is done loading and rendered, too.
```

Comment:
```suggestion
      // root's children are done loading and rendered, too.
```

Reply:
There's only one in this case though, right?

Reply:
Oh sorry, I got confused by the lines below this (`REQUIRE(parentB3DM.getChildren().size() == 4);`) and thought the comment was referring to this check. Nevermind 🙂